The job **`aarch64-gnu-llvm-21-1`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/27868289336/job/82475938795) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/82475938795) [(plain)](https://github.com/rust-lang/rust/commit/e61ac0e6abbd828429ebcd3346c8bb410d11423c/checks/82475938795/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
test [ui] tests/ui/zero-sized/zero-sized-btreemap-insert.rs ... ok

failures:

---- [ui] tests/ui/asm/s390x/bad-reg.rs#s390x stdout ----
Saved the actual stderr to `/checkout/obj/build/aarch64-unknown-linux-gnu/test/ui/asm/s390x/bad-reg.s390x/bad-reg.s390x.stderr`
diff of stderr:

279 error: register class `vreg` requires the `vector` target feature
280   --> $DIR/bad-reg.rs:71:18
281    |
- LL |         asm!("", in("v0") v); // requires vector & asm_experimental_reg
+ LL |         asm!("", in("v0") v);
283    |                  ^^^^^^^^^^
284 
285 error: register class `vreg` requires the `vector` target feature

286   --> $DIR/bad-reg.rs:73:18
287    |
- LL |         asm!("", out("v0") v); // requires vector & asm_experimental_reg
+ LL |         asm!("", out("v0") v);
289    |                  ^^^^^^^^^^^
290 
291 error: register class `vreg` requires the `vector` target feature

292   --> $DIR/bad-reg.rs:75:18
293    |
- LL |         asm!("", in("v0") x); // requires vector & asm_experimental_reg
+ LL |         asm!("", in("v0") x);
295    |                  ^^^^^^^^^^
296 
297 error: register class `vreg` requires the `vector` target feature

298   --> $DIR/bad-reg.rs:77:18
299    |
- LL |         asm!("", out("v0") x); // requires vector & asm_experimental_reg
+ LL |         asm!("", out("v0") x);
301    |                  ^^^^^^^^^^^
302 
303 error: register class `vreg` requires the `vector` target feature

315 error: register class `vreg` requires the `vector` target feature
316   --> $DIR/bad-reg.rs:85:26
317    |
- LL |         asm!("/* {} */", in(vreg) v); // requires vector & asm_experimental_reg
+ LL |         asm!("/* {} */", in(vreg) v);
319    |                          ^^^^^^^^^^
320 
321 error: register class `vreg` requires the `vector` target feature

322   --> $DIR/bad-reg.rs:87:26
323    |
- LL |         asm!("/* {} */", in(vreg) x); // requires vector & asm_experimental_reg
+ LL |         asm!("/* {} */", in(vreg) x);
325    |                          ^^^^^^^^^^
326 
327 error: register class `vreg` requires the `vector` target feature

333 error: register class `vreg` requires the `vector` target feature
334   --> $DIR/bad-reg.rs:92:26
335    |
- LL |         asm!("/* {} */", out(vreg) _); // requires vector & asm_experimental_reg
+ LL |         asm!("/* {} */", out(vreg) _);
337    |                          ^^^^^^^^^^^
338 
339 error: type `i32` cannot be used with this register class


The actual stderr differed from the expected stderr
To update references, rerun the tests and pass the `--bless` flag
To only update this specific test, also pass `--test-args asm/s390x/bad-reg.rs`

error in revision `s390x`: 1 errors occurred comparing output.
status: exit status: 1
command: env -u RUSTC_LOG_COLOR RUSTC_ICE="0" RUST_BACKTRACE="short" "/checkout/obj/build/aarch64-unknown-linux-gnu/stage2/bin/rustc" "/checkout/tests/ui/asm/s390x/bad-reg.rs" "-Zthreads=1" "-Zsimulate-remapped-rust-src-base=/rustc/FAKE_PREFIX" "-Ztranslate-remapped-path-to-local-path=no" "-Z" "ignore-directory-in-diagnostics-source-blocks=/cargo" "-Z" "ignore-directory-in-diagnostics-source-blocks=/checkout/vendor" "--sysroot" "/checkout/obj/build/aarch64-unknown-linux-gnu/stage2" "--cfg" "s390x" "--check-cfg" "cfg(test,FALSE,s390x,s390x_vector)" "--error-format" "json" "--json" "future-incompat" "-Ccodegen-units=1" "-Zui-testing" "-Zdeduplicate-diagnostics=no" "-Zwrite-long-types-to-disk=no" "-Cstrip=debuginfo" "--emit" "metadata" "-C" "prefer-dynamic" "--out-dir" "/checkout/obj/build/aarch64-unknown-linux-gnu/test/ui/asm/s390x/bad-reg.s390x" "-A" "unused" "-W" "unused_attributes" "-A" "internal_features" "-A" "incomplete_features" "-A" "unused_parens" "-A" "unused_braces" "-Crpath" "-Cdebuginfo=0" "-Lnative=/checkout/obj/build/aarch64-unknown-linux-gnu/native/rust-test-helpers" "-Cpanic=abort" "-Cforce-unwind-tables=yes" "--target" "s390x-unknown-linux-gnu" "-C" "target-feature=-vector" "--extern" "minicore=/checkout/obj/build/aarch64-unknown-linux-gnu/test/ui/asm/s390x/bad-reg.s390x/libminicore.rlib"
stdout: none
--- stderr -------------------------------
error: invalid register `r11`: The frame pointer cannot be used as an operand for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:28:18
   |
LL |         asm!("", out("r11") _);
   |                  ^^^^^^^^^^^^

error: invalid register `r15`: The stack pointer cannot be used as an operand for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:30:18
   |
LL |         asm!("", out("r15") _);
   |                  ^^^^^^^^^^^^

error: invalid register `c0`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:32:18
   |
LL |         asm!("", out("c0") _);
   |                  ^^^^^^^^^^^

error: invalid register `c1`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:34:18
   |
LL |         asm!("", out("c1") _);
   |                  ^^^^^^^^^^^

error: invalid register `c2`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:36:18
   |
LL |         asm!("", out("c2") _);
   |                  ^^^^^^^^^^^

error: invalid register `c3`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:38:18
   |
LL |         asm!("", out("c3") _);
   |                  ^^^^^^^^^^^

error: invalid register `c4`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:40:18
   |
LL |         asm!("", out("c4") _);
   |                  ^^^^^^^^^^^

error: invalid register `c5`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:42:18
   |
LL |         asm!("", out("c5") _);
   |                  ^^^^^^^^^^^

error: invalid register `c6`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:44:18
   |
LL |         asm!("", out("c6") _);
   |                  ^^^^^^^^^^^

error: invalid register `c7`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:46:18
   |
LL |         asm!("", out("c7") _);
   |                  ^^^^^^^^^^^

error: invalid register `c8`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:48:18
   |
LL |         asm!("", out("c8") _);
   |                  ^^^^^^^^^^^

error: invalid register `c9`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:50:18
   |
LL |         asm!("", out("c9") _);
   |                  ^^^^^^^^^^^

error: invalid register `c10`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:52:18
   |
LL |         asm!("", out("c10") _);
   |                  ^^^^^^^^^^^^

error: invalid register `c11`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:54:18
   |
LL |         asm!("", out("c11") _);
   |                  ^^^^^^^^^^^^

error: invalid register `c12`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:56:18
   |
LL |         asm!("", out("c12") _);
   |                  ^^^^^^^^^^^^

error: invalid register `c13`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:58:18
   |
LL |         asm!("", out("c13") _);
   |                  ^^^^^^^^^^^^

error: invalid register `c14`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:60:18
   |
LL |         asm!("", out("c14") _);
   |                  ^^^^^^^^^^^^

error: invalid register `c15`: control registers are reserved by the kernel and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:62:18
   |
LL |         asm!("", out("c15") _);
   |                  ^^^^^^^^^^^^

error: invalid register `a0`: a0 and a1 are reserved for system use and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:64:18
   |
LL |         asm!("", out("a0") _);
   |                  ^^^^^^^^^^^

error: invalid register `a1`: a0 and a1 are reserved for system use and cannot be used as operands for inline asm
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:66:18
   |
LL |         asm!("", out("a1") _);
   |                  ^^^^^^^^^^^

error: register class `areg` can only be used as a clobber, not as an input or output
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:98:18
   |
LL |         asm!("", in("a2") x);
   |                  ^^^^^^^^^^

error: register class `areg` can only be used as a clobber, not as an input or output
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:101:18
   |
LL |         asm!("", out("a2") x);
   |                  ^^^^^^^^^^^

error: register class `areg` can only be used as a clobber, not as an input or output
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:104:26
   |
LL |         asm!("/* {} */", in(areg) x);
   |                          ^^^^^^^^^^

error: register class `areg` can only be used as a clobber, not as an input or output
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:107:26
   |
LL |         asm!("/* {} */", out(areg) _);
   |                          ^^^^^^^^^^^

error: register `f0` conflicts with register `v0`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:112:31
   |
LL |         asm!("", out("v0") _, out("f0") _);
   |                  -----------  ^^^^^^^^^^^ register `f0`
   |                  |
   |                  register `v0`

error: register `f1` conflicts with register `v1`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:114:31
   |
LL |         asm!("", out("v1") _, out("f1") _);
   |                  -----------  ^^^^^^^^^^^ register `f1`
   |                  |
   |                  register `v1`

error: register `f2` conflicts with register `v2`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:116:31
   |
LL |         asm!("", out("v2") _, out("f2") _);
   |                  -----------  ^^^^^^^^^^^ register `f2`
   |                  |
   |                  register `v2`

error: register `f3` conflicts with register `v3`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:118:31
   |
LL |         asm!("", out("v3") _, out("f3") _);
   |                  -----------  ^^^^^^^^^^^ register `f3`
   |                  |
   |                  register `v3`

error: register `f4` conflicts with register `v4`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:120:31
   |
LL |         asm!("", out("v4") _, out("f4") _);
   |                  -----------  ^^^^^^^^^^^ register `f4`
   |                  |
   |                  register `v4`

error: register `f5` conflicts with register `v5`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:122:31
   |
LL |         asm!("", out("v5") _, out("f5") _);
   |                  -----------  ^^^^^^^^^^^ register `f5`
   |                  |
   |                  register `v5`

error: register `f6` conflicts with register `v6`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:124:31
   |
LL |         asm!("", out("v6") _, out("f6") _);
   |                  -----------  ^^^^^^^^^^^ register `f6`
   |                  |
   |                  register `v6`

error: register `f7` conflicts with register `v7`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:126:31
   |
LL |         asm!("", out("v7") _, out("f7") _);
   |                  -----------  ^^^^^^^^^^^ register `f7`
   |                  |
   |                  register `v7`

error: register `f8` conflicts with register `v8`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:128:31
   |
LL |         asm!("", out("v8") _, out("f8") _);
   |                  -----------  ^^^^^^^^^^^ register `f8`
   |                  |
   |                  register `v8`

error: register `f9` conflicts with register `v9`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:130:31
   |
LL |         asm!("", out("v9") _, out("f9") _);
   |                  -----------  ^^^^^^^^^^^ register `f9`
   |                  |
   |                  register `v9`

error: register `f10` conflicts with register `v10`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:132:32
   |
LL |         asm!("", out("v10") _, out("f10") _);
   |                  ------------  ^^^^^^^^^^^^ register `f10`
   |                  |
   |                  register `v10`

error: register `f11` conflicts with register `v11`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:134:32
   |
LL |         asm!("", out("v11") _, out("f11") _);
   |                  ------------  ^^^^^^^^^^^^ register `f11`
   |                  |
   |                  register `v11`

error: register `f12` conflicts with register `v12`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:136:32
   |
LL |         asm!("", out("v12") _, out("f12") _);
   |                  ------------  ^^^^^^^^^^^^ register `f12`
   |                  |
   |                  register `v12`

error: register `f13` conflicts with register `v13`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:138:32
   |
LL |         asm!("", out("v13") _, out("f13") _);
   |                  ------------  ^^^^^^^^^^^^ register `f13`
   |                  |
   |                  register `v13`

error: register `f14` conflicts with register `v14`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:140:32
   |
LL |         asm!("", out("v14") _, out("f14") _);
   |                  ------------  ^^^^^^^^^^^^ register `f14`
   |                  |
   |                  register `v14`

error: register `f15` conflicts with register `v15`
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:142:32
   |
LL |         asm!("", out("v15") _, out("f15") _);
   |                  ------------  ^^^^^^^^^^^^ register `f15`
   |                  |
   |                  register `v15`

error: invalid register `f16`: unknown register
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:145:32
   |
LL |         asm!("", out("v16") _, out("f16") _);
   |                                ^^^^^^^^^^^^

error: register class `vreg` requires the `vector` target feature
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:71:18
   |
LL |         asm!("", in("v0") v);
   |                  ^^^^^^^^^^

error: register class `vreg` requires the `vector` target feature
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:73:18
   |
LL |         asm!("", out("v0") v);
   |                  ^^^^^^^^^^^

error: register class `vreg` requires the `vector` target feature
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:75:18
   |
LL |         asm!("", in("v0") x);
   |                  ^^^^^^^^^^

error: register class `vreg` requires the `vector` target feature
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:77:18
   |
LL |         asm!("", out("v0") x);
   |                  ^^^^^^^^^^^

error: register class `vreg` requires the `vector` target feature
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:79:18
   |
LL |         asm!("", in("v0") b);
   |                  ^^^^^^^^^^

error: register class `vreg` requires the `vector` target feature
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:82:18
   |
LL |         asm!("", out("v0") b);
   |                  ^^^^^^^^^^^

error: register class `vreg` requires the `vector` target feature
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:85:26
   |
LL |         asm!("/* {} */", in(vreg) v);
   |                          ^^^^^^^^^^

error: register class `vreg` requires the `vector` target feature
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:87:26
   |
LL |         asm!("/* {} */", in(vreg) x);
   |                          ^^^^^^^^^^

error: register class `vreg` requires the `vector` target feature
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:89:26
   |
LL |         asm!("/* {} */", in(vreg) b);
   |                          ^^^^^^^^^^

error: register class `vreg` requires the `vector` target feature
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:92:26
   |
LL |         asm!("/* {} */", out(vreg) _);
   |                          ^^^^^^^^^^^

error: type `i32` cannot be used with this register class
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:98:27
   |
LL |         asm!("", in("a2") x);
   |                           ^
   |
   = note: register class `areg` supports these types: 

error: type `i32` cannot be used with this register class
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:101:28
   |
LL |         asm!("", out("a2") x);
   |                            ^
   |
   = note: register class `areg` supports these types: 

error: type `i32` cannot be used with this register class
##[error]  --> /checkout/tests/ui/asm/s390x/bad-reg.rs:104:35
   |
LL |         asm!("/* {} */", in(areg) x);
   |                                   ^
   |
   = note: register class `areg` supports these types: 

error: aborting due to 54 previous errors
```

</details>

--- a/src/doc/unstable-book/src/language-features/asm-experimental-reg.md
+++ b/src/doc/unstable-book/src/language-features/asm-experimental-reg.md
@@ -1,4 +1,4 @@
-# `asm_experimental_arch`
+# `asm_experimental_reg`
 
 The tracking issue for this feature is: [#133416]
 

--- a/tests/ui/asm/s390x/bad-reg.rs
+++ b/tests/ui/asm/s390x/bad-reg.rs
@@ -68,13 +68,13 @@ fn f() {
 
         // vreg
         asm!("", out("v0") _); // always ok
-        asm!("", in("v0") v); // requires vector & asm_experimental_reg
+        asm!("", in("v0") v);
         //[s390x]~^ ERROR register class `vreg` requires the `vector` target feature
-        asm!("", out("v0") v); // requires vector & asm_experimental_reg
+        asm!("", out("v0") v);
         //[s390x]~^ ERROR register class `vreg` requires the `vector` target feature
-        asm!("", in("v0") x); // requires vector & asm_experimental_reg
+        asm!("", in("v0") x);
         //[s390x]~^ ERROR register class `vreg` requires the `vector` target feature
-        asm!("", out("v0") x); // requires vector & asm_experimental_reg
+        asm!("", out("v0") x);
         //[s390x]~^ ERROR register class `vreg` requires the `vector` target feature
         asm!("", in("v0") b);
         //[s390x]~^ ERROR register class `vreg` requires the `vector` target feature
@@ -82,14 +82,14 @@ fn f() {
         asm!("", out("v0") b);
         //[s390x]~^ ERROR register class `vreg` requires the `vector` target feature
         //[s390x_vector]~^^ ERROR type `u8` cannot be used with this register class
-        asm!("/* {} */", in(vreg) v); // requires vector & asm_experimental_reg
+        asm!("/* {} */", in(vreg) v);
         //[s390x]~^ ERROR register class `vreg` requires the `vector` target feature
-        asm!("/* {} */", in(vreg) x); // requires vector & asm_experimental_reg
+        asm!("/* {} */", in(vreg) x);
         //[s390x]~^ ERROR register class `vreg` requires the `vector` target feature
         asm!("/* {} */", in(vreg) b);
         //[s390x]~^ ERROR register class `vreg` requires the `vector` target feature
         //[s390x_vector]~^^ ERROR type `u8` cannot be used with this register class
-        asm!("/* {} */", out(vreg) _); // requires vector & asm_experimental_reg
+        asm!("/* {} */", out(vreg) _);
         //[s390x]~^ ERROR register class `vreg` requires the `vector` target feature
 
         // Clobber-only registers

--- a/tests/ui/asm/s390x/bad-reg.s390x.stderr
+++ b/tests/ui/asm/s390x/bad-reg.s390x.stderr
@@ -279,25 +279,25 @@ LL |         asm!("", out("v16") _, out("f16") _);
 error: register class `vreg` requires the `vector` target feature
   --> $DIR/bad-reg.rs:71:18
    |
-LL |         asm!("", in("v0") v); // requires vector & asm_experimental_reg
+LL |         asm!("", in("v0") v);
    |                  ^^^^^^^^^^
 
 error: register class `vreg` requires the `vector` target feature
   --> $DIR/bad-reg.rs:73:18
    |
-LL |         asm!("", out("v0") v); // requires vector & asm_experimental_reg
+LL |         asm!("", out("v0") v);
    |                  ^^^^^^^^^^^
 
 error: register class `vreg` requires the `vector` target feature
   --> $DIR/bad-reg.rs:75:18
    |
-LL |         asm!("", in("v0") x); // requires vector & asm_experimental_reg
+LL |         asm!("", in("v0") x);
    |                  ^^^^^^^^^^
 
 error: register class `vreg` requires the `vector` target feature
   --> $DIR/bad-reg.rs:77:18
    |
-LL |         asm!("", out("v0") x); // requires vector & asm_experimental_reg
+LL |         asm!("", out("v0") x);
    |                  ^^^^^^^^^^^
 
 error: register class `vreg` requires the `vector` target feature
@@ -315,13 +315,13 @@ LL |         asm!("", out("v0") b);
 error: register class `vreg` requires the `vector` target feature
   --> $DIR/bad-reg.rs:85:26
    |
-LL |         asm!("/* {} */", in(vreg) v); // requires vector & asm_experimental_reg
+LL |         asm!("/* {} */", in(vreg) v);
    |                          ^^^^^^^^^^
 
 error: register class `vreg` requires the `vector` target feature
   --> $DIR/bad-reg.rs:87:26
    |
-LL |         asm!("/* {} */", in(vreg) x); // requires vector & asm_experimental_reg
+LL |         asm!("/* {} */", in(vreg) x);
    |                          ^^^^^^^^^^
 
 error: register class `vreg` requires the `vector` target feature
@@ -333,7 +333,7 @@ LL |         asm!("/* {} */", in(vreg) b);
 error: register class `vreg` requires the `vector` target feature
   --> $DIR/bad-reg.rs:92:26
    |
-LL |         asm!("/* {} */", out(vreg) _); // requires vector & asm_experimental_reg
+LL |         asm!("/* {} */", out(vreg) _);
    |                          ^^^^^^^^^^^
 
 error: type `i32` cannot be used with this register class

